odoolint/tests/test_convert_file.py

```diff
@@ -178,6 +181,7 @@ def test_70_ref_unreachable_csv(self):
         self.create_unreachable('base.group_user')
         self.create_imd('security/ir.model.access.csv', 'data', 2, 1)
 
+    @unittest.skipIf(ODOO_PATCHED is True, "Odoo is patched with str2bool")
```

Comment:
Hi @moylop260 
I tried that way as you said and didn't work.

Also, in my last change I used `import unittest2 as unittest` in that case skip the test method without the use of the class `unittest.TestCase`
However, with any of the pushed ways the test is **skipped** but **only in my local container**
![bd-using-odoolint](https://cloud.githubusercontent.com/assets/11741384/18291393/1a0ee58e-744d-11e6-8f41-590d46c5dab6.png)
![skipped test 80](https://cloud.githubusercontent.com/assets/11741384/18291399/228f1954-744d-11e6-9a84-3c7bd70d5f93.png)

I checked the log in runbot for the PR dummy in lodigroup: https://github.com/Vauxoo/lodigroup/pull/606
And is not taking the change:
![odoo_lint_isolated](https://cloud.githubusercontent.com/assets/11741384/18291476/75a627cc-744d-11e6-981f-5e7c854103a7.png)
Is there a possibility that the problem is the `odoo_lint_isolated` instead of `odoolint`? Or I have to do a dummy commit changing something else?